devstudio/installer/category.xml

```diff
@@ -0,0 +1,18 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<site>
+	<description>
+		JBoss Developer Studio Integration Stack Installer
+	</description>
+
+	<!-- copy IU's from JBDS Core Installer -->
+	<feature id="org.jboss.tools.bpel.feature" />
+	<feature id="org.testng.eclipse" />
+	<bundle id="com.jboss.devstudio.core.package" />
```

Comment:
if that is the case then we'll be rebuilding/repacking the JBDS parts - running the risk of changing something we did not expect.

The intent/idea was that the IS installer would *only* be changing the IS part so we all could be fairly certain nothing surprising changed.

If this is necessary then it starts making sense that we just have one installer to build and release which over time adds more and more IS bits in ...but now doing a nightly build of the installer becomes tricky because of IS being out-of-sync.

i.e. JBDS 9 core builds an installer with CR1 in it.
JBDS-IS would take that installer, but since they are only on Beta2 this newer version of installer now contains *old* bits.

Thats an annoying problem ;/